### ISSUE TITLE
Evaluate other toggles to see if any should be turned on automatically with TOGGLE SCREENREADER.

### ISSUE THREAD

**Issue by luciensadi** [luciensadi/AwakeMUD#332]
Ex: fightgag, movegag, compact, menugag...
